app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = @user.comments.new(post: @post, text: params[:text])
+    if @comment.save
+      redirect_to user_posts_path(@user, @post)
+    else
+      render 'new'
+    end
+  end
+end
```

Comment:
- [OPTIONAL] KIndly, it is a good approach to use the params method.  Lesson link about the strong parameters: [Link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)

- Strong Params allow developers to specify in the controller which parameters are accepted and used. It use in many Rails applications to increase the security of data sent through forms.  eg:

```ruby
  private 
  def comment_params
    params.require(:comment).permit(:text)
  end
```